### PR TITLE
[Documentation] Local dictionary Data which are not supported float and byte updated in the note

### PR DESCRIPTION
**Pull request by sgururajshetty**

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/750/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9016/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/948/

File: docs/configuration-parameters.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ This section provides the details of all the configurations required for the Car
...
+| carbon.load.min.size.enabled | false | During Data Loading, CarbonData would divide the number of files among the available executors to parallelize the loading operation. When the input data files are very small, this action causes to generate many small carbondata files. This configuration determines whether to enable node minumun input data size allocation strategy for data loading. It will make sure that the nodes load the minimum amount of data there by reducing number of carbondata files.**NOTE:** This configuration is useful if the size of the input data files are very small, like 1MB to 256MB.Refer to ***load_min_size_inmb*** to configure the minimum size to be considered for splitting files among executors. |
```
Comment (sraghunandan) [inline docs/configuration-parameters.md]: give space after full stops

Comment (sraghunandan) [inline docs/configuration-parameters.md]: give space after full stops

File: docs/ddl-of-carbondata.md [hunk 2]
```diff
@@ -234,7 +234,8 @@ CarbonData DDL statements are documented here,which includes:
       * TIMESTAMP
       * DATE
       * BOOLEAN
-   
+	  * FLOAT
```
Comment (sraghunandan) [inline docs/ddl-of-carbondata.md]: indentation is wrong, change the settings to replace tabs with spaces in your editor

File: docs/configuration-parameters.md (hunk 1 above)
Comment (sraghunandan) [inline docs/configuration-parameters.md]: give space after full stops

Comment (sraghunandan) [inline docs/configuration-parameters.md]: add a space after full stops.

Comment (sraghunandan) [inline docs/configuration-parameters.md]: add space after full stops

File: docs/ddl-of-carbondata.md (hunk 2 above)
Comment (sraghunandan) [inline docs/ddl-of-carbondata.md]: correct the indentation. Add settings to change tab to spaces in your editor

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1049/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1050/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1051/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1265/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9317/

**Comment by sgururajshetty** [comment]
@sraghunandan kindly review and help me to merge my changes

**Comment by sgururajshetty** [comment]
@sraghunandan kindly review and help me to merge my changes

**Comment by sgururajshetty** [comment]
@sraghunandan kindly review

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1307/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1518/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1512/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9770/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1722/

**Comment by sraghunandan** [comment]
LGTM

**Pull request closed without merging**